doc/reference_measure.xml

```diff
@@ -820,6 +820,64 @@ SELECT degrees(ST_Azimuth(ST_Point(25, 45), ST_Point(75, 100))) AS degA_B,
 		</refsection>
 
   </refentry>
+  
+  	<refentry id="ST_Angle">
+		<refnamediv>
+		  <refname>ST_Angle</refname>
+
+		  <refpurpose>Returns the angle in radian measured clokwise between 3 points, or the angle measured clockwise between 2 vectors.</refpurpose>
```

Comment:
"clokwise" -> "clockwise" (first occurrence)

If I may suggest also keeping the "refpurpose" text small (may omit radian and clockwise, leaving it specified in the "description")